receiver.go

```diff
@@ -45,14 +47,30 @@ func IsForcePush(hook *HookInfo) (bool, error) {
 	return base != hook.OldRev, nil
 }
 
+func (r *Receiver) CheckAllowedBranch(hook *HookInfo) error {
+	if r.MasterOnly { // for BC
+		r.AllowedBranches = append(r.AllowedBranches, "refs/heads/master")
```

Comment:
Figured comparing to the actual names, saved some parsing. But let me know if you'd like another _API_ with straight up branch names like `[]string{"master","main","etc"}` instead.